docs/src/main/asciidoc/azure-support-overview-concept.adoc

```diff
@@ -0,0 +1,117 @@
+////
+This document is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="azure-support-overview-concept"]
+= Quarkus on Microsoft Azure cloud platform
+include::_attributes.adoc[]
+:categories: cloud
+
+Quarkus provides support and capabilities to help you to build cloud-native applications that run on Microsoft Azure. Quarkus also provides support for building serverless Java functions by using Microsoft Azure Functions.
+You can deploy the container image of a Quarkus application that you have developed locally to the Microsoft Azure cloud platform.
+You can also entirely build, host, and run a Quarkus application on Azure by using some of the key developer services for Azure.
+
+== Microsoft Azure
+
+link:https://learn.microsoft.com/en-us/azure/?product=popular[Microsoft Azure] is a versatile and flexible cloud platform, which you can use to host your Quarkus applications.
+Azure supports various programming languages, complex architectures and frameworks, including Java and Quarkus.
+Azure also fully supports the Apache Maven Java development tool.
+You can use several link:https://learn.microsoft.com/en-us/azure/developer/intro/azure-developer-key-services[Azure key services for developers] to build and extend your Quarkus applications.
+You can also integrate and deploy Quarkus with the serverless environment of link:https://learn.microsoft.com/en-us/azure/azure-functions/[Azure Functions].
+
+Microsoft Azure also provides tools and services that integrate with Quarkus to make it easier to develop, deploy, and manage your Java applications on its cloud platform. For example:
+
+* link:[Azure DevOps] provides CI/CD pipelines for building and deploying Quarkus applications.
+* link:[Azure Monitor] provides capabilities for you to use to monitor and troubleshoot these applications.
+
+== Quarkus extensions for Microsoft Azure
+
+Quarkus provides the following extensions to make it easier to integrate your existing Quarkus applications with Microsoft Azure:
+
+
+* `quarkus-azure-functions`
+* `quarkus-azure-functions-http`
+
+// What about these extensions? Should we mention them?
+//* link:[Azure Functions]
+//* link:[Azure Storage]
+//* link:[Azure Key Vault]
+//* link:[Azure Service Bus]
+
+== Deploying Quarkus applications on Microsoft Azure
+
+There are several ways in which you can deploy a Quarkus application on link:https://learn.microsoft.com/en-us/azure/?product=popular[Microsoft Azure].
+
+The general process involves building and deploying a Docker container image of your Quarkus application to an Azure container service and configuring the networking and security settings as needed.
+
+=== Microsoft Azure deployment tools
+
+You can run Quarkus applications on Azure by deploying them as a Docker container and  using a combination of some of the following Microsoft integration tools:
```

Comment:
Hmm - we shouldn't need to push azure specific features first. 

Shouldn't we show the ways quarkus can deploy that requires zero or very little setup first ?

Ie. Deploying to Azure container service should be doable 100% with our builtin quarkus features.

Reply:
@maxandersen - Thanks very much Max for the initial feedback. This is really helpful :+1:  Sorry for the slow reply. I was on PTO. 

I'll rework the content, based on your initial feedback above :arrow_up: 

The aim of the concept topic was to provide some context to some of the other existing doc resources that are available for deploying and developing on Azure, which will also be restructured into diatazis content types as part of this issue. If I understand correctly, we need to make the distinction between deploying info and developing/integrating services with Azure Functions. Would it make more sense to have 2 separate concept pages to introduce Quarks deployment on Azure versus Quarkus development/integration with Azure Functions? Thanks